src/main/scala/mesosphere/marathon/tasks/PortsMatcher.scala

```diff
@@ -96,7 +99,7 @@ class PortsMatcher(
     takeEnoughPortsOrNone(expectedSize = numberOfPorts) {
       shuffledAvailablePorts.map(Some(_))
     } orElse {
-      log.info(s"Couldn't find $numberOfPorts ports in offer for app [${app.id}]")
+      log.info(s"Offer ${offer.getId.getValue}. Couldn't find $numberOfPorts ports in offer for app [${app.id}]")
```

Comment:
Why do some interpolated values have surrounding square brackets and others do not?  I like the square brackets because they make grepping logs easier, and it also looks obvious when the empty string is interpolated.